lib/LANraragi/Model/Stats.pm

```diff
@@ -199,12 +181,13 @@ sub build_tag_stats {
 }
 
 sub compute_content_size {
+    my $redis_db = LANraragi::Model::Config->get_redis;
 
-    #Get size of archive folder
-    my $dirname = LANraragi::Model::Config->get_userdir;
-    my $size    = 0;
-
-    find( sub { $size += -s if -f }, $dirname );
+    my @keys = $redis_db->keys('????????????????????????????????????????');
+    my $size = 0;
+    foreach my $id (@keys) {
```

Comment:
It'd probably be a fair bit faster here to use WATCH/MULTI to get the arcsize values for each ID in one redis transaction instead of opening/closing a db connection each time.

Reply:
Oops, missed that every call to get_redis created a new connection.